src/dune

```diff
@@ -24,9 +24,17 @@
       (apron -> apronAnalysis.apron.ml)
       (-> apronAnalysis.no-apron.ml)
     )
-    (select apronPriv.ml from
-      (apron -> apronPriv.apron.ml)
-      (-> apronPriv.no-apron.ml)
```

Comment:
Does the `ApronPriv` (renamed `RelationPriv`) really not require apron anymore? (We do currently not check this in the CI, so one has to verify this locally in a switch where apron is not installed).

Reply:
The unlocked workflow, which checks without apron installed as well, can manually be run from the Actions tab. Although since the PR is coming from a fork, @MartinWehking needs to do that himself on his fork.

Reply:
unlocked throws no errors in runs without apron. What's still quite odd though is that the 'build'  process fails in every run using Ocaml version 4.10.x with the message `Error: Unbound value equal` in `affineEqualityDomain` (line 108) but that's a different issue.

Reply:
unlocked works without errors anymore.  9470349ad42bc98999fe7c16c88ce80edba022b2 and b287752600c0f9949d97a000da12fcbd26377511